### DISCUSSION TITLE
what is this error mean?

### DISCUSSION THREAD

**Discussion by kururu-abdo** [firebase/flutterfire#10201] [Q&A]
[SmsRetrieverHelper] SMS verification code request failed: unknown status code: 17499 Error code:39

**Comment by MazinHashim** [comment]
because of concept of signup quota which means: limit the number of new Email/Password and Anonymous accounts that your application can create in a day from a single IP address.
This happens because the signup quota limit exceeded, you can increase it from Authentication Settings -> Sign-up quota
